env_setup/env_setup.sh

```diff
@@ -0,0 +1,101 @@
+}
+
+
+# echo "Do you to perform pre_setup?"
+# echo -e "!!!! PRE_SETUP:\nThis will make python3 and pip3 as default versions\n\t+\nit will install virtualenv and virtualenvwrapper\n!!!!\n"
+# echo -e "-- Enter the number for your selection and press enter!"
+# select yn in "Yes" "No"; do
+#     case $yn in
+#         Yes ) bash $(echo $PWD)/pre_setup.sh; break;;
+#         No ) echo -e "You selected \"NO\" so make sure that you have python3 and pip3 installed otherwise script will fail"; exit;;
+#     esac
+# done
+
+
+add_virtualenvwrppaer_env_var() {
+    FILE=$1
+    CONTENTS=$2
+    echo -e "$CONTENTS" >> $FILE
+}
+
+install_virt_pckgs
+
+### update environement vars
+curr_term=$(echo $SHELL)
+PROFILE_FILE=""
+echo "$curr_term"
+if [ "$curr_term" == "/bin/zsh" ]; then
+    PROFILE_FILE="$HOME/.zshrc"
+else
+    PROFILE_FILE="$HOME/.bash_profile"
+fi
+
+Contents=""
+if [ "$(echo $VIRTUALENV_PYTHON)" != "$(which python)" ]; then
+    Contents+="export VIRTUALENV_PYTHON=/usr/local/bin/python3 \n"
+fi
+if [ "$(echo $WORKON_HOME)" != "$HOME/Envs" ]; then
+    Contents+="export WORKON_HOME=~/Envs \n"
+    mkdir -p $HOME/Envs
+fi
+
+if [ "$Contents" != "" ];then
+    Contents+="source /usr/local/bin/virtualenvwrapper.sh\n"
+    add_virtualenvwrppaer_env_var $PROFILE_FILE "\n\n$Contents"
+    # add_virtualenvwrppaer_env_var $a "\n\n$Contents"
+fi
+
+
+### virtualenv setup
+echo "Do you wish to continue and allow the script to create virtualenv for you?"
+echo -e "-- Enter the number for your selection and press enter!"
+select yn in "Yes" "No"; do
+    case $yn in
+        Yes ) echo -e "Creating virtualenv"; break;;
+        No ) echo -e "You selected \"NO\" so you have to create the virtualenv and install all the dependecies and libs with the required versions"; exit;;
+    esac
+done
+# Creating and activating virtual env
+export VIRTUALENV_PYTHON=/usr/local/bin/python3
+export WORKON_HOME=~/Envs
+source /usr/local/bin/virtualenvwrapper.sh
+mkvirtualenv yomeal >&2 > /dev/null
+echo "$COL_GREEN -- new virtualenv named \"yomeal\" has been created$RST --"
+workon yomeal
+# Installing dependencies/libs
+requirements_file="$(echo $PWD)/requirements.txt"
+if [ -f "$requirements_file" ]; then
+    echo "$requirements_file exist; so installing the requirements"
+    pip3 install -r $requirements_file >&2 > /dev/null
+fi
+
+echo -e "\n\n$COL_GREEN!!!!!!!!!!!!!!!!!!! Completed !!!!!!!!!!!!!!!!!!!$RST\n\n"
+echo -e "$COL_RED Confirm Folloiwng steps before moving further$RST"
+echo -e "$COL_YELLOW - Make Sure your virtualenv has been created. You can confirm by going into \'~/Envs/\' and confirm if \"yomeal\" dir exists."
+echo -e "$COL_YELLOW - Execute \'source ~/.zsh_rc\' or \'source ~/.bash_profile\' or close&reopen terminal"
+echo -e "$COL_YELLOW - Goto your project directory and confimr if you are able to access the virtual env by executing \'workon yomeal\'"
+echo -e "$COL_YELLOW - If you require pre_setup.sh or not"
```

Comment:
Sure you want to override `COL_YELLOW` var in above all statements.

Reply:
Yes.